wandb/sdk/service/server.py

```diff
@@ -126,9 +128,13 @@ def serve(self) -> None:
         self._inform_used_ports(grpc_port=grpc_port, sock_port=sock_port)
         setproctitle = wandb.util.get_optional_module("setproctitle")
         if setproctitle:
-            service_ver = 0
+            service_ver = 2
+            pid = str(self._pid or 0)
+            transport = "s" if sock_port else "g"
             port = grpc_port or sock_port or 0
-            service_id = f"{service_ver}-{port}"
+            # this format is similar to wandb_manager token but it purely informative now
+            # (consider unifying this in the future)
+            service_id = f"{service_ver}-{pid}-{transport}-{port}"
```

Comment:
yeah... probably need some utility to keep it consistent

Reply:
wandb_manager: _ManagerToken

Reply:
Ah good point! :)